src/components/NodeToolbar/NodeToolbar.js

```diff
@@ -258,26 +260,38 @@ class NodeToolbar extends React.Component<Props, State> {
   }
 
   renderInlineWheelchairAccessibilityEditor() {
```

Comment:
@saryekad seems to have removed this code while you have extended it: https://github.com/sozialhelden/wheelmap-frontend/pull/247#pullrequestreview-384237305 - could you sync this?

Reply:
I am still waiting for @saryekad to go into master before rebasing. The whole pr will likely change.

Reply:
I was waiting, will rebase today :)

Reply:
Done, was easy enough :)